docs/en/operations/settings/settings.md

```diff
@@ -307,7 +307,51 @@ Disabled by default.
 
 ## input_format_tsv_enum_as_number {#settings-input_format_tsv_enum_as_number}
 
-For TSV input format switches to parsing enum values as enum ids.
+Enables or disables parsing enum values as enum ids for TSV input format.
+
+Possible values:
+
+-   0 — Enum values are parsed as values.
+-   1 — Enum values are parsed as enum IDs
+
+Default value: 0.
+
+**Example**
+
+Consider the table:
+
+```sql
+CREATE TABLE table_with_enum_column_for_tsv_insert (Id Int32,Value Enum('first' = 1, 'second' = 2)) ENGINE=Memory() FORMAT TabSeparatedRaw;
```

Comment:
Small comment: it would be better to remove `FORMAT TabSeparatedRaw`, it's meaningless in this example.